### PR TITLE
ORM#4-SideMenu-Automated

### LINKED ISSUE #4: Side Menu Testing
- Login and Dashboard 
- Default dashboard is selected 
- Search function 
- Side menu close and open function
- Logo Click redirection.

### PR DESCRIPTION
**Pull request by adityasinghtads**
Side Menu Automated.

### CODE REVIEW

File: src/test/java/in/automateORM/sideMenuTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+	}
+	
+	@Test(priority=1)
+	public void sideMenuOpen() {
+		
+		login.performLogin("Admin", "admin123", driver);
+		WebElement openSideMenu =  driver.findElement(By.xpath("//*[@id=\"app\"]/div[1]/div[1]/aside/nav/div[2]/div/div/button/i"));
+		openSideMenu.click();
+		openSideMenu.click();
```
Comment (selAnsh) [inline src/test/java/in/automateORM/sideMenuTest.java:36]: logic for two times click ?

**Review by selAnsh — CHANGES_REQUESTED** [review]
Is there any provision to use a function generic for before and after method ?
Perhaps can be defined in main class

File: src/test/java/in/automateORM/sideMenuTest.java (hunk 1 above)
Comment (adityasinghtads) [reply src/test/java/in/automateORM/sideMenuTest.java]: This method is to test, weather the side menu bar is opening and closing, 
have updated the function.

**Comment by adityasinghtads** [comment]
> Is there any provision to use a function generic for before and after method ? Perhaps can be defined in main class

Need to check, will create a future task to design and incoperate this.

**Review by selAnsh — APPROVED** [review]

**Pull request merged**